nav2_navfn_planner/src/navfn_planner.cpp

```diff
@@ -102,8 +100,20 @@ NavfnPlanner::computePathToPose(const nav2_tasks::ComputePathToPoseCommand::Shar
       planner_ = std::make_unique<NavFn>(costmap_.metadata.size_x, costmap_.metadata.size_y);
     }
 
+	  // Get the current pose from the robot
+    auto start = std::make_shared<geometry_msgs::msg::PoseWithCovarianceStamped>();
+
+    if (!robot_->getCurrentPose(start)) {
+      RCLCPP_ERROR(get_logger(), "Current robot pose is not available.");
+      return TaskStatus::FAILED;
+    }
+
+    RCLCPP_INFO(get_logger(), "Attempting to a find path from (%.2f, %.2f) to "
```

Comment:
Should we change this to RCLCPP_DEBUG? Otherwise it will get printed every second.

Reply:
@crdelsey Yes, I'll update.